scene/animation/tween.cpp

```diff
@@ -821,17 +821,20 @@ bool Tween::remove(Object *p_object, String p_key) {
 		call_deferred("remove", p_object, p_key);
 		return true;
 	}
+	List<List<InterpolateData>::Element *> for_removal;
 	for(List<InterpolateData>::Element *E=interpolates.front();E;E=E->next()) {
 
 		InterpolateData& data = E->get();
 		Object *object = ObjectDB::get_instance(data.id);
 		if(object == NULL)
 			continue;
-		if(object == p_object && data.key == p_key) {
-			interpolates.erase(E);
-			return true;
+		if(object == p_object && (data.key == p_key || p_key == "")) {
+			for_removal.push_back(E);
```

Comment:
I think it's not a good idea.
Other methods do not break out of the loop on first match.
Probably they don't because you can add more than one tween with the same property (which may be confusing in my opinion).

This code still starts a tween in 2.1:
`var tween = get_node("Tween")`
`var label = get_node("Label")`
`tween.interpolate_property(label, "rect/scale", other params...)`
`tween.interpolate_property(label, "rect/scale", other params...)`
`tween.remove(label, "rect/scale")`
`tween.start()`
But when you replace `tween.remove()` with `tween.stop()` it won't start a tween because there's no break in the stop code.

Probably it would be the best to have only one tween for the property and replace old tween with new one if there's old one already. Or change other three methods so they also break out of the loop.

Maybe we need more opinions on this.
I vote to use version without break even if it's breaking change, still I think it's more like bug fixing than breaking...